hazelcast-build-utils/src/test/resources/manifest.mf

```diff
@@ -252,7 +252,7 @@ Export-Package: com.hazelcast.cluster.impl;uses:="com.hazelcast.loggin
  memory.impl,javax.management,javax.management.openmbean,sun.misc,com.
  hazelcast.spi.properties,com.hazelcast.client.impl,com.hazelcast.conf
  ig,com.hazelcast.spi.impl.executionservice,com.hazelcast.internal.clu
- ster.impl,com.hazelcast.cluster.memberselector,com.hazelcast.memory,c
+ ster.impl,com.hazelcast.cluster.memberselector,com.hazelcast.internal.memory,c
```

Comment:
This is probably irrelevant, but shouldn't the test file `manifest.mf` be either: updated to say `version="4.0.SNAPSHOT"`, or: kept intact to use the old package structure?

Reply:
Good question. I would say updated but it's more of a wild guess as I have no real experience or knowledge with manifest.mf